board/variscite/am62px_var_som/am62px_var_som.env

```diff
@@ -9,7 +9,7 @@ main_cpsw0_qsgmii_phyinit=false
 
 name_kern=Image
 console=ttyS0,115200n8
-args_all=run ramsize_check; setenv optargs ${optargs} earlycon=ns16550a,mmio32,0x02800000 ${cma_size} cma_name=linux,cma
+args_all=i2c dev 2; i2c probe; run ramsize_check; setenv optargs ${optargs} earlycon=ns16550a,mmio32,0x02800000 ${cma_size} cma_name=linux,cma
```

Comment:
@FrancescoFerraro, I suppose the above change (adding `i2c dev 2; i2c probe;`) is just inherited by the previous JIG changes for AM62, correct ? 
@eranmati, @aviadh80, being this just printing the addresses in us on I2C0, is this wanted ?